lib/query-utils.js

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2020 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+
+const cleanQuery = (query) => {
+  const regex = /\([\sa-zA-Z0-9,\:\"\'\{\}\$\!]+\)/g
+
+  const innerRegex = /(\:)[\sa-zA-Z0-9\"\'\$\!]+([^\{]?[,\s\}\)])/g
+
+  const matches = query.match(regex)
+  
+  if (matches) {
+    matches.forEach((match) => {
+      const newArgBlock = match.replace(innerRegex, '$1' + '***' + '$2')
```

Comment:
Not sure if anything more standard is done as far as replacement text like `***`. It seems fine and obvious to me. On the SQL side, we do `?` but that is a little more related to how SQL is typically written.